### PR TITLE
Medium p 2

### PR DESCRIPTION
**Pull request by agavriluta**
Task example :
image

### CODE REVIEW

File: HardProject/src/main/java/org/example/App.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package org.example;
+
+/**
+ * Hello world!
+ *
+ */
+public class App
```
Comment (ppanasencoGD) [inline HardProject/src/main/java/org/example/App.java:7]: Why do you have hard task package?

File: MediumProject/src/main/java/org/example/stage2/Searching.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
+package org.example.stage2;
+
+import java.util.*;
+
+public class Searching {
+
+    private static List<String[]> listOfData(){
+        System.out.println("Enter the number of people:");
+        Scanner scan = new Scanner(System.in);
+        int size = scan.nextInt();
```
Comment (ppanasencoGD) [inline MediumProject/src/main/java/org/example/stage2/Searching.java:9]: if input will be -3?

File: MediumProject/src/main/java/org/example/stage2/Searching.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            data.add(input[i].split(" "));
+        }
+        return data;
+    }
+    public static void searchQueries(){
+        List<String[]> data = listOfData();
+        System.out.println("Enter the number of search queries:");
+        Scanner scan = new Scanner(System.in);
+        Scanner scanner = new Scanner(System.in);
+        int count = scan.nextInt();
```
Comment (ppanasencoGD) [inline MediumProject/src/main/java/org/example/stage2/Searching.java]: please give more descriptive variable names
I dont understand what is responsible for : temp, temp2, count, a , res,

File: MediumProject/src/main/java/org/example/stage2/Searching.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        System.out.println();
+        for (int i = 0; i < size; i++) {
+            data.add(input[i].split(" "));
+        }
+        return data;
+    }
+    public static void searchQueries(){
+        List<String[]> data = listOfData();
+        System.out.println("Enter the number of search queries:");
+        Scanner scan = new Scanner(System.in);
```
Comment (ppanasencoGD) [inline MediumProject/src/main/java/org/example/stage2/Searching.java]: is it possible to use one scanner for all inputs?

File: MediumProject/src/main/java/org/example/stage2/Searching.java [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+        scan.nextLine();
+        System.out.println("Enter all people:");
+        List<String[]> data= new ArrayList<>();
+        String[] input = new String[size];
+        for (int i = 0; i < input.length; i++) {
+            input[i] = scan.nextLine();
+        }
+        System.out.println();
+        for (int i = 0; i < size; i++) {
+            data.add(input[i].split(" "));
```
Comment (ppanasencoGD) [inline MediumProject/src/main/java/org/example/stage2/Searching.java]: convert array in a list and later use list.containsIgnoseCase util method to search word

File: MediumProject/src/main/java/org/example/stage2/Searching.java (hunk 2 above)
Comment (agavriluta) [reply MediumProject/src/main/java/org/example/stage2/Searching.java:9]: in the task it's impossible 
they don't have such case

File: MediumProject/src/main/java/org/example/stage2/Searching.java [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+        String searchWord = scan.nextLine();
+        foundUsers.clear();
+        for (String row: addUsersToGeneralList()) {
+            if (row.contains(searchWord)) {
+                foundUsers.add(row);
+            }
+        }
+    }
+
+    private static void printResult(Set<String> res) {
```
Comment (ppanasencoGD) [inline MediumProject/src/main/java/org/example/stage2/Searching.java:41]: res - not readable name

**Review by ppanasencoGD — APPROVED** [review]
Please squash before merge

File: MediumProject/src/main/java/org/example/stage2/Searching.java (hunk 6 above)
Comment (agavriluta) [reply MediumProject/src/main/java/org/example/stage2/Searching.java:41]: done

**Pull request merged**